retrofit-http-logging/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Sure, if you want it. I'm just trying to stop the torrent of SO, issues, and questions on how to log (since there's some tricky things and some not-so-tricky things).